endpoint_info.go

```diff
@@ -227,7 +232,17 @@ func (ep *endpoint) DriverInfo() (map[string]interface{}, error) {
 		return nil, fmt.Errorf("failed to get driver info: %v", err)
 	}
 
-	return driver.EndpointOperInfo(n.ID(), ep.ID())
+	epInfo, err := driver.EndpointOperInfo(n.ID(), ep.ID())
+	if err != nil {
+		return nil, err
+	}
+
+	if epInfo != nil {
+		epInfo["GW_INFO"] = gwDriverInfo
+		return epInfo, nil
+	}
+
+	return gwDriverInfo, nil
```

Comment:
This is because in case of adding gateway in the sandbox there are two endpoints in the sandbox. Docker daemon only has information for one endpoint. Windows driver relies on getting endpoint driverinfo to actually connect endpoint to containers. There is no other way for me to get driver info for both the gateway endpoint and overlay endpoint from docker daemon. Is there something you can suggest?